### PR TITLE
Add Automatic Scenario Assignments to Compass UI

### PR DESCRIPTION
**Pull request by NickyMateev**
# Overview

This pull request adds Automatic Scenario Assignments to the Scenario details page of Compass UI.

There was an existing bug where if you wanted to remove a Runtime from a scenario the UI would allow that, but the backend would reject the request ultimately resulting in a false perception that the runtime was removed. Actually Compass UI on refresh would show you that the runtime is still in the scenario.

The bug happened when there were existing automatic scenario assignments for the specific scenario which are responsible for the labelling of the runtime. Compass UI would try to manually **setRuntimeLabel** when unassigning the runtime from the scenario, but that wouldn't work as the automatic scenario assignment would re-label the runtime again.

Just so that such confusions and bugs don't persist this PR proposes adding any existing Automatic Scenario Assignment entity to it's respective Scenario in the Scenario Details UI page like so:

image

If admin tries to remove Runtime from scenario and existing automatic scenario assignment exists for it an error alert is shown at the top of his screen:

image

This way if an admin wants to unassign a runtime from a scenario, he'd have to remove the associated scenario assignment.

Additional things done part of this PR:
- Forbid unassigning runtime from scenario if existing automatic scenario assignment for it exists [Scenario Details view]
- Forbid unassigning runtime from scenario if existing automatic scenario assignment for it exists [Runtime Details view]
- Tests have been enabled for the Compass UI
- **jest** snapshot test added for the new _ScenarioDetails_ component

### CODE REVIEW

**Review by KirilKabakchiev — APPROVED** [review]
Cant comment much on the UI code, but the new UI addition works fine locally :) i think it is a great idea to have ASAs management in the UI

**Comment by KirilKabakchiev** [comment]
Another thought - this allows deletion of ASAs, what about creation ?

**Comment by NickyMateev** [comment]
@KirilKabakchiev let's not get to ahead of ourselves with the creation :D

**Comment by NickyMateev** [comment]
Will be moved to https://github.com/kyma-incubator/compass-console

File: components/console/compass/src/components/Runtimes/RuntimeDetails/RuntimeScenarios/RuntimeScenarios.component.js [hunk 1]
```diff
@@ -31,12 +44,31 @@ export default function RuntimeScenarios({
   ];
 
   async function deactivateScenario(entry) {
+    const scenarioAssignments =
+      fetchedScenarioAssignments.automaticScenarioAssignments.data;
     const scenarioName = entry.scenario;
 
+    let canDeactivate = true;
+    scenarioAssignments.forEach(function(asa) {
```
[the remark this answers is not in the document]
Comment (NickyMateev) [reply components/console/compass/src/components/Runtimes/RuntimeDetails/RuntimeScenarios/RuntimeScenarios.component.js:52]: Nice one! 👍

**Pull request closed without merging**